docs/dashboards/datagrid-component.md

```diff
@@ -0,0 +1,104 @@
+DataGrid Component
+===
+
+The DataGrid Component can be placed inside a data cell to allow the data visualization in the form of an editable data grid.
+
+<iframe style="width: 100%; height: 470px; border: none;" src="https://www.highcharts.com/samples/nonav/dashboards/datagrid-component/datagrid-options" allow="fullscreen"></iframe>
+
+
+## How to start
+
+1. To use the DataGrid Component in Dashboards, you need to import the appropriate modules.
+    ```html
+    <script src="https://code.highcharts.com/dashboards/dashboards.js"></script>
+    <script src="https://code.highcharts.com/dashboards/datagrid.js"></script>
+    <script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+    ```
+
+    Alternatively, you can also use NPM packages (see: [Installation](https://www.highcharts.com/docs/dashboards/installation)) and import to connect them to the Dashboards.
+    ```ts
+    import * as Dashboards from '@highcharts/dashboards';
+    import DataGrid from '@highcharts/dashboards/es-modules/DataGrid/DataGrid';
+    import DataGridPlugin from '@highcharts/dashboards/es-modules/Dashboards/Plugins/DataGridPlugin';
+
+    DataGridPlugin.custom.connectDataGrid(DataGrid);
+    Dashboards.PluginHandler.addPlugin(DataGridPlugin);
+    ```
+
+2. You also need to import the styles.
+    ```css
+    @import url("https://code.highcharts.com/css/dashboards.css");
+    @import url("https://code.highcharts.com/css/datagrid.css");
+    ```
+
+3. After loading the necessary files, define a cell using a unique identifier for the cell, e.g. `dashboard-col-0`. This cell will be used to place the component in the dashboard.
+    ```js
+    gui: {
+        layouts: [{
+            id: 'layout-1',
+            rows: [{
+                cells: [{
+                    id: 'dashboard-col-0'
+                }]
+            }]
+        }]
+    }
+    ```
+
+4. You will also need some data that you could display in your DataGrid component. For this purpose, you can, for example, define a connector that, in this case, downloads CSV data from the example URL.
+    ```js
+    dataPool: {
+        connectors: [{
+            id: 'data',
+            type: 'CSV',
+            options: {
+                csvURL: 'https://demo-live-data.highcharts.com/updating-set.csv'
+            }
+        }]
+    }
+    ```
+    Click [here](https://www.highcharts.com/docs/dashboards/data-handling) to read more about the data handling.
+
+5. Once you have the data and where to place your component, you can define it as below. In the `cell` option, we're passing the ID of the cell, which we defined in point 3, and the ID of the connector with the data in the `connector.id` option (point 4). In order for the component to be created as a DataGrid, it was set with the `type` option to `'DataGrid'`.
```

Comment:
What if I don't have a connector or I don't want to use it? How to create a simple grid without?

Reply:
Scheduled after bug fix with the [dataTable](https://api.highcharts.com/dashboards/#interfaces/DataGrid_DataGridOptions.DataGridOptions-1#dataTable) option, which doesn't work for the DataGrid Component.